src/components/UserSelector.tsx

```diff
@@ -1,24 +1,15 @@
-import React, { useContext, useEffect, useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import classNames from 'classnames';
-import { UserContext } from './UsersContext';
-import { User } from '../types/User';
+import { useAppDispatch, useAppSelector } from '../app/hooks';
```

Comment:
Consider importing 'getUsers' from '../features/user/usersSlice' instead of '../api/users', as it is the convention to import actions from the slice file.